user-guide/docs/index.md

```diff
@@ -6,6 +6,7 @@ Any natural hazards researcher or practitioner that wants an environment to stor
 
 1. [**Request a user account**](https://www.designsafe-ci.org/account/register/)
 2. You will be redirected to the TACC account registration page.
+    a. It's possible that your email provider will either block or mark as spam/junk an email TACC sends with a link to confirm your email address after you submit the registration form. If so, send email to helpATdesignsafe-ci.org letting us know you did not receive the email.      
```

Comment:
It could be nice if that worked, but _nested_ [lists in Markdown](https://www.markdownguide.org/basic-syntax/#ordered-lists) also use numbers. I'll add CSS to make sure it appears as a letter.

```suggestion
    1. It's possible that your email provider will either block or mark as spam/junk an email TACC sends with a link to confirm your email address after you submit the registration form. If so, send email to helpATdesignsafe-ci.org letting us know you did not receive the email.      
```